### PR TITLE
New way requesting vas for booking api 4 e dz zmx a

### PR DESCRIPTION
**Pull request by cyberq**

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/4EDzZmxA](https://trello.com/c/4EDzZmxA)

File: api/booking/api.raml [hunk 1]
```diff
@@ -19,6 +19,38 @@ documentation:
 - title: Encoding
   content: |
     Booking endpoints will assume all requests are UTF-8 encoded
+      
+- title: A new way of requesting additional services
```
Comment (pankajagrawal16) [inline api/booking/api.raml]: image


I think we should make bullet points and say why they should use only new way.

Ex: All new VASes will be added only on new way., Old way will be decommissioned in some time etc. More flexible etc to make changes when new way is added.

File: api/booking/api.raml [hunk 2]
```diff
@@ -19,6 +19,38 @@ documentation:
...
+            <mobile>12345</mobile>
+          </additionalService>
+          <additionalService>
+            <id>DANGEROUS_GOODS</id>
+            <mobile>777777</mobile>
+            <unNumber>ABC</unNumber>
+          </additionalService>
+       </additionalServices>
+
+       The old way of requesting additional servies:
```
Comment (pankajagrawal16) [inline api/booking/api.raml]: Is there a way to show there two separately in markup and old one with a strike through or something.

Comment (pankajagrawal16) [reply api/booking/api.raml]: Also may be this section should be updated as well. http://developer.test.bring.com/api/booking/#book-shipments

Comment (cyberq) [reply api/booking/api.raml]: can strike through the heading

**Pull request merged**